libhb/enc_qsv.c

```diff
@@ -692,12 +692,18 @@ int encqsvInit(hb_work_object_t *w, hb_job_t *job)
     // set and enable colorimetry (video signal information)
     switch (job->color_matrix_code)
     {
-        case 4:
+        case 5:
```

Comment:
Since we are shifting the meaning of the codes here, maybe we should take the opportunity to define or enum these rather than using magic numbers.  

It might also be a good idea to set the "custom" value to something we won't change in the future, although that means making "custom" 1 and redefining all the values or making custom a large enough value that it will never need to change.